### PR TITLE
add /test/e2e-aws-opearator job to cluster-kube-apiserver-operator

### PR DESCRIPTION
**Pull request by sanchezl**
/hold
tests are not merged yet

### CODE REVIEW

**Comment by sanchezl** [comment]
https://github.com/openshift/cluster-kube-apiserver-operator/pull/333

**Comment by sanchezl** [comment]
The latest version of the generator touched every job file, so holding until I figure out why.

File: ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml [hunk 1]
```diff
@@ -49,6 +49,15 @@ tests:
   openshift_installer:
     cluster_profile: aws
 
+- as: e2e-aws-operator
+  commands: |
+    mkdir -p /tmp/artifacts
+    go get -u github.com/openshift/release/tools/gotest2junit
+    make check-e2e-operator JUNITFILE=/tmp/artifacts/unit_report.xml
```
Comment (tnozicka) [inline ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml]: make test-e2e JUNITFILE=/tmp/artifacts/unit_report.xml

Comment (sanchezl) [reply ci-operator/config/openshift/cluster-kube-apiserver-operator/openshift-cluster-kube-apiserver-operator-master.yaml]: Done.

**Comment by sanchezl** [comment]
> The latest version of the generator touched every job file, so holding until I figure out why.

After pulling a new version of the generator, everything works again.

**Comment by tnozicka** [comment]
/approve
/lgtm

/hold 
this needs to go first https://github.com/openshift/cluster-kube-apiserver-operator/pull/333

**Comment by tnozicka** [comment]
the other PR is at a phase where it is close to landing unholding so we can test it out there
/hold cancel

**Pull request merged**